lib/javascript/manage-food-table.js

```diff
@@ -1,23 +1,53 @@
-// I should see a table of all my foods, with Name, Calories and a delete icon for each food
-// <tr>
-// <td>This is a name.</td>
-// <td>This is a calorie.</td>
-// <td>This is a delete icon.</td>
-// <tr> -->
-// tableclass is manage-foods
-
-// https://quantify-this-api.herokuapp.com/api/v1/foods
-// return $.getJSON(`${host}/posts`)
 $ = require('jquery');
 
 $(document).ready(function(){
   getAllFoods();
+  $('.new-food-form').on('submit', function(){
+    event.preventDefault();
+    var foodName = $('input[name="new-food-name"]').val();
+    var foodCalories = $('input[name="new-food-calories"]').val();
+    postNewFood({food: {name: foodName, calories: foodCalories}});
+    $('.manage-foods > tbody > tr:first').after(`<tr><td>${foodName}</td><td>${foodCalories}</td><td><div class="delete-icon"></div></td><tr>`);
+
+  });
+  $('.manage-foods').on('click', '.delete-icon', function(event){
+    var foodId = this.classList[1].split('-').pop();
+    deleteFood(foodId);
+    $(this.parentElement.parentElement).remove()
+  })
 });
 
 const getAllFoods = function(){
   $.getJSON('https://quantify-this-api.herokuapp.com/api/v1/foods', function(result) {
-    $.each(result, function(i, field){
-      $('table').append(`<tr><td>${field.name}</td><td>${field.calories}</td><td><div class="delete-icon"></div></td><tr>`);
-    })
-  })
+    sortedResult = result.sort(function(a,b){
+      return b.id-a.id
+    });
+    $.each(sortedResult, function(i, field){
+      $('table').append(`<tr><td>${field.name}</td><td>${field.calories}</td><td><div class="delete-icon delete-${field.id}"></div></td><tr>`);
```

Comment:
the html here looks very similar to what's on line 10. could we extract this out so we can reuse it rather than re-writing it?